### ISSUE TITLE
NSPrivacyAccessedAPICategoryUserDefaults missing from PrivacyInfo file

### ISSUE THREAD

**Issue by mcg95** [queueit/ios-webui-sdk#33]
https://github.com/queueit/ios-webui-sdk/blob/0940f8de79311ad0f769ee20c5ea169052e81752/QueueITLib/QueueCache.m#L23

Since NSUserDefaults is being used, the PrivacyInfo file must also contain NSPrivacyAccessedAPITypes - NSPrivacyAccessedAPIType - NSPrivacyAccessedAPICategoryUserDefaults - NSPrivacyAccessedAPITypeReasons and the relevant [reason](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api) (eg: CA92.1) for the API being used. 

```	
<key>NSPrivacyAccessedAPITypes</key>
	<array>
		<dict>
			<key>NSPrivacyAccessedAPIType</key>
			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
			<key>NSPrivacyAccessedAPITypeReasons</key>
			<array>
				<string>CA92.1</string>
			</array>
		</dict>
	</array>
```

I used this [tool](https://github.com/crasowas/app_store_required_privacy_manifest_analyser) to scan the project to find the issues/

Tools Output related to QueueIt:
```
Analyzing QueueITLibrary ...
💡 Found privacy manifest file(s): 1
[0] /.../ios/Pods/QueueITLibrary/QueueITLib/PrivacyInfo.xcprivacy
API usage analysis result(s): 1
[0] NSPrivacyAccessedAPICategoryUserDefaults:UserDefaults,NSUserDefaults:/.../ios/Pods/QueueITLibrary/QueueITLib/QueueCache.m
🛠️  Descriptions for the following required API reason(s) may be missing: 1
[0] NSPrivacyAccessedAPICategoryUserDefaults
```

**Comment by bret-queueit** [comment]
Hi @mcg95!
Thanks for reaching out to us.
QueueCache files have been deprecated and are not being used anymore. We have updated our repo by deleting the files, so you should not have this issue anymore.
Thanks!

**Comment by mcg95** [comment]
Deletion of the QueueCache files have not fixed the issue as we are pointing at 3.4.1 on our podfile and the 3.4.1 tag still contains the QueueCache files that were deleted.

**Comment by bret-queueit** [comment]
Hi [mcg95](https://github.com/mcg95) !

Sorry for the inconvenience. We have released version 3.4.2 with the update.
Thanks!
